storage.go

```diff
@@ -172,23 +175,23 @@ func (s *Storage) createLink(ctx context.Context, path string, target string, op
 	}
 
 	var sm ObjectSystemMetadata
-	if v := aws.StringValue(output.ServerSideEncryption); v != "" {
```

Comment:
@Xuanwo 
For this kind of conversion, i just use string() to replace it.
So I have to use aws.ToString() to replace it .am i right?